--- src/statusicon.c.str	2009-03-15 12:39:27.000000000 +0100
+++ src/statusicon.c	2009-03-15 12:39:40.000000000 +0100
@@ -212,7 +212,7 @@
         icon_path = g_build_filename(DATADIR, ICON_PATH PACKAGE ".svg", NULL);
         if (!(pixbuf = gdk_pixbuf_new_from_file(icon_path, &error))) {
                 status_icon = NULL;
-                g_warning(error->message);
+                g_warning("%s",error->message);
                 g_warning("Failed to load status icon '%s'", icon_path);
                 return;
         }
